src/corona_hakab_model/generation/matrix_generator.py

```diff
@@ -24,6 +25,36 @@
 from project_structure import OUTPUT_FOLDER
 
 
+class AgentConnections(NamedTuple):
+    daily_connections: List = set()
+    weekly_connections: List = set()
```

Comment:
type hinting doesn't match the used type.
change type hinting to Set.

Reply:
Did you assume my type?!?!?

Reply:
oh right, changed one but not the other